content/components/button.mdx

```diff
@@ -231,6 +231,14 @@ Buttons allow users to initiate an action or command when clicked or tapped. The
 
 ## Best practices
 
+### Primary and outline button usage
+
+Limit primary button usage. Only use one primary button on the page, whenever possible, to indicate its emphasis relative to other actions. Primary buttons have top priority in visual hierarchy, and using too many of them on a single view dilutes their effectiveness.
+
+Use outline buttons independently. Outline buttons are mainly used for secondary actions such as filtering. Don't use in conjunction or place in close proximity of other button styles
```

Comment:
```suggestion
```

We're kinda quietly deprecating outline buttons. I feel like we could just delete this line

Reply:
Removed!